content/blog/eks-v3-release/index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Pulumi EKS Provider Version 3.0.0"
+date: 2024-10-04T17:32:03+02:00
+draft: true
+meta_desc: "Release of the next version of the Pulumi Provider for AWS EKS"
+meta_image: meta.png
+authors:
+    - florian-stadler
+tags:
+    - aws
+    - eks
+    - release
+---
+
+The latest major release of the Pulumi EKS Provider is available now!
+
+AWS recently announced the deprecation of two features used by default in Pulumi EKS: the aws-auth ConfigMap and the AL2 operating system. Pulumi EKS v3 addresses these deprecations, enhances the maintainability of the provider, and aligns it with EKS best practices.
+This release delivers significant improvements in flexibility and security, introducing new features to enhance your Kubernetes experience on AWS.
```

Comment:
Maybe lead with the positive impact for users, rather than deprecations that we had to deal with which sounds more like just maintenance?